### DISCUSSION TITLE
Accesibility issue?

### DISCUSSION THREAD

**Discussion by AlvaroStream** [ianyh/Amethyst#1539] [Q&A] [answered]
Hello, 

I haven't changed anything from yesterday to today (no update in MacOS, no update of Amethyst) but now when I try to set the layout with the shortcuts . MacOs shows me this: 
![Screenshot 2023-08-23 at 13 35 28](https://github.com/ianyh/Amethyst/assets/102966649/90932956-bfa5-4537-9056-9a34b4fcafce)
Even if I have the Accesibility active:
![Screenshot 2023-08-23 at 13 35 53](https://github.com/ianyh/Amethyst/assets/102966649/aaa626cd-2ad1-4ef0-8d1c-01a66d7ef1af)

Not sure how to track it neither solve it.

Ps.- Thanks for such an amazing software that make MacOS bearable (a little).

**Comment by ianyh** [comment, marked as answer]
@AlvaroStream you can try the following:
1. Quit Amethyst.
2. Remove Amethyst from the list of allowed applications. (Click on the row, and use the minus button at the bottom.)
3. Launch Amethyst again and give permissions again.

**Comment by AlvaroStream** [comment]
Thanks, yes it auto resolved like that.
